ch_pipeline/processing/daily.py

```diff
@@ -622,7 +622,75 @@
         output_name: "ringmap_intercyl_hpf_{{tag}}.zarr.zip"
         remove: true
 
+    # Take the variance over frequency for the XX pol of the delay-filtered
+    # ringmap
+    - type: draco.analysis.transform.Reduce
+      in: ringmap_int_hpf
+      out: ringmap_reduced_freq
+      params:
+        axes: ["freq"]
+        datasets: ["map"]
+        op: "var"
+        selections: {"pol":[0, 3]}
+        generate_weight: true
+        apply_weight: false
```

Comment:
Making sure params agree with `Reduce` task:
```suggestion
        generate_mask: true
        apply_mask: false
```

Reply:
Good catch, thanks. I might actually have to change this again after reworking the `Reduce` task a bit